### PR TITLE
Refactor commands to use Cobra

### PR DESCRIPTION
**Pull request by baderbuddy**
Changed the interface to use Cobra commands.

### CODE REVIEW

File: cmd/polaris/root.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+func init() {
+	// Flags
+	rootCmd.PersistentFlags().StringVarP(&configPath, "config", "c", "", "Location of Polaris configuration file.")
+	rootCmd.PersistentFlags().BoolVarP(&disallowExemptions, "disallow-exemptions", "", false, "Disallow any exemptions from configuration file.")
+	rootCmd.PersistentFlags().StringVarP(&logLevel, "log-level", "", logrus.InfoLevel.String(), "Logrus log level.")
+	flag.Parse()
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+}
+var c conf.Configuration
```
Comment (rbren) [inline cmd/polaris/root.go]: Since this is global now, can we give it a better name?

File: cmd/polaris/version.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Prints the current version of the tool.",
+	Long:  `Prints the current version.`,
```
Comment (rbren) [inline cmd/polaris/version.go]: I think you've got short/long reversed :)

File: docs/usage.md [hunk 3]
```diff
@@ -119,8 +119,8 @@ kubectl apply -f https://github.com/fairwindsops/polaris/releases/latest/downloa
 
 ### Helm
 ```bash
-helm repo add reactiveops-stable https://charts.reactiveops.com/stable
-helm upgrade --install polaris reactiveops-stable/polaris --namespace polaris \
+helm repo add fairwindsops-stable https://charts.fairwinds.com/stable
```
Comment (rbren) [inline docs/usage.md]: `s/fairwindsops/fairwinds`, below too

File: main.go [hunk 4]
```diff
@@ -15,279 +15,17 @@
...
-	"sigs.k8s.io/controller-runtime/pkg/webhook"
-	"sigs.k8s.io/yaml"
+	"github.com/fairwindsops/polaris/cmd/polaris"
 )
 
 const (
 	// Version represents the current release version of Polaris
 	Version = "0.6.0"
+	// TODO make sure this gets set.
+	commit = "n/a"
```
Comment (rbren) [inline main.go]: What's this for?

Comment (baderbuddy) [reply main.go]: Copied from Goldilocks and not needed and can be safely removed

Comment (baderbuddy) [reply main.go]: Well, it was used for the version output, maybe it still would be valuable

**Comment by baderbuddy** [comment]
Tests are passing and ready for another look

File: main.go (hunk 4 above)
Comment (rbren) [reply main.go]: I'd say let's get rid of it. There's no way to check the commit hash in without changing the commit hash, so I can't imagine what this is meant for.

**Pull request merged**